src/jdk.internal.le/share/classes/jdk/internal/org/jline/terminal/impl/ffm/CLibrary.java

```diff
@@ -0,0 +1,1250 @@
+    private static int XTABS;
+    private static int BSDLY;
+    private static int BS0;
+    private static int BS1;
+    private static int VTDLY;
+    private static int VT0;
+    private static int VT1;
+    private static int CBAUD;
+    private static int B0;
+    private static int B50;
+    private static int B75;
+    private static int B110;
+    private static int B134;
+    private static int B150;
+    private static int B200;
+    private static int B300;
+    private static int B600;
+    private static int B1200;
+    private static int B1800;
+    private static int B2400;
+    private static int B4800;
+    private static int B9600;
+    private static int B19200;
+    private static int B38400;
+    private static int EXTA;
+    private static int EXTB;
+    private static int OFDEL;
+    private static int ONOEOT;
+    private static final int OCRNL;
+    private static int ONOCR;
+    private static final int ONLRET;
+    private static int OFILL;
+
+    private static int CIGNORE;
+    private static int CSIZE;
+    private static final int CS5;
+    private static final int CS6;
+    private static final int CS7;
+    private static final int CS8;
+    private static final int CSTOPB;
+    private static final int CREAD;
+    private static final int PARENB;
+    private static final int PARODD;
+    private static final int HUPCL;
+    private static final int CLOCAL;
+    private static int CCTS_OFLOW;
+    private static int CRTS_IFLOW;
+    private static int CDTR_IFLOW;
+    private static int CDSR_OFLOW;
+    private static int CCAR_OFLOW;
+
+    private static final int ECHOKE;
+    private static final int ECHOE;
+    private static final int ECHOK;
+    private static final int ECHO;
+    private static final int ECHONL;
+    private static final int ECHOPRT;
+    private static final int ECHOCTL;
+    private static final int ISIG;
+    private static final int ICANON;
+    private static int XCASE;
+    private static int ALTWERASE;
+    private static final int IEXTEN;
+    private static final int EXTPROC;
+    private static final int TOSTOP;
+    private static final int FLUSHO;
+    private static int NOKERNINFO;
+    private static final int PENDIN;
+    private static final int NOFLSH;
+
+    static {
+        String osName = System.getProperty("os.name");
+        if (osName.startsWith("Linux")) {
+            String arch = System.getProperty("os.arch");
+            boolean isMipsPpcOrSparc = arch.equals("mips")
+                    || arch.equals("mips64")
+                    || arch.equals("mipsel")
+                    || arch.equals("mips64el")
+                    || arch.startsWith("ppc")
+                    || arch.startsWith("sparc");
+            TIOCGWINSZ = isMipsPpcOrSparc ? 0x40087468 : 0x00005413;
+            TIOCSWINSZ = isMipsPpcOrSparc ? 0x80087467 : 0x00005414;
+
+            TCSANOW = 0x0;
+            TCSADRAIN = 0x1;
+            TCSAFLUSH = 0x2;
+
+            VINTR = 0;
```

Comment:
I'm just curious: How did you arrive at these values? They seem like they are generated somehow. (I guess you did not just make them up and write them down here). Can that process be documented? Or saved as a script in case they ever need to be updated?